scripts/maintenance/auto-cleanup.sh

```
@@ -0,0 +1,236 @@
+RED='\033[0;31m'
+BLUE='\033[0;34m'
+NC='\033[0m'
+
+echo -e "${BLUE}🧹 Dotfiles 自動クリーンアップ${NC}"
+echo "━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━"
+echo "📁 対象ディレクトリ: $DOTFILES_DIR"
+echo "🔄 モード: $([ "$DRY_RUN" == "true" ] && echo "DRY-RUN (確認のみ)" || echo "実行モード")"
+echo "📊 開始時刻: $(date '+%Y-%m-%d %H:%M:%S')"
+echo ""
+
+cd "$DOTFILES_DIR"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)